src/main/resources/application.yml

```diff
@@ -4,7 +4,14 @@ server:
     context-path: /notifications
 
 application:
+  domain: ${APP_DOMAIN:}
+  email:
+      sender: ${EMAIL_SENDER}
```

Comment:
I assume bounce-backs will go to this mailbox - will we have access to this? (And/or the ability to configure an alarm if it has mail, that would be cool!)

Reply:
Bounces and complaints can be routed to SNS/SQS, then we can have either a simple alarm on the SQS queue or something sat at the end of that queue to automate the follow-up.